code-samples/serving/hello-world/helloworld-csharp/README.md

```diff
@@ -19,16 +19,16 @@ cd knative-docs/code-samples/serving/hello-world/helloworld-csharp
   [Install Knative Serving](https://knative.dev/docs/install/serving/install-serving-with-yaml).
 - [Docker](https://www.docker.com) installed and running on your local machine,
   and a Docker Hub account configured (we'll use it for a container registry).
-- You have installed [.NET Core SDK 3.1](https://www.microsoft.com/net/core).
+- You have installed [.NET Core SDK 6.0](https://www.microsoft.com/net/).
 
 ## Recreating the sample code
 
 1. First, make sure you have
-   [.NET Core SDK 3.1](https://www.microsoft.com/net/core) installed:
+   [.NET SDK 6.0](https://www.microsoft.com/net/) installed:
 
    ```bash
    dotnet --version
-   3.1.100
+   6.0.101
    ```
 
 1. From the console, create a new empty web project using the dotnet command:
```

Comment:
Need to update the instructions to match the new example that only has one file `Program.cs`